### ISSUE TITLE
2 Errors

### ISSUE THREAD

**Issue by GoodGuy8** [rohanpillai20/Image-Classification-by-Keras-and-Tensorflow#1]
**Error 1:**
WARNING:tensorflow:From <ipython-input-1-8b8315d34fe4>:41: calling argmax (from tensorflow.python.ops.math_ops) with dimension is deprecated and will be removed in a future version.
Instructions for updating:
Use the `axis` argument instead

**Error 2:**
saver.save(session, 'C:\\

Does not save anything, so when calling Prediction.py it says:
Exception: File models/trained_model.meta does not exist. 

**Error 3:**
Warning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
